### PR TITLE
20 aboutpage

### LINKED ISSUE #20: implement the about page
implement https://www.wheatleyvodka.com/about-wheatley/

split this task into multiple, check #66 

this page has various components that need to be defined:
- [x] straight from the source section
- [x] faq-replace block for Q&A (why limestone water etc)
- [x] video playback

### PR DESCRIPTION
**Pull request by RemoLiechti**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #20 

Test URLs:
- Before: https://main--sazerac-wheatleyvodka--hlxsites.hlx.page/about-wheatley
- After: https://20-aboutpage--sazerac-wheatleyvodka--hlxsites.hlx.page/about-wheatley

### CODE REVIEW

File: blocks/embed/embed.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+main .embed {
```
Comment (RemoLiechti) [inline blocks/embed/embed.css:1]: copied from boilerplate

File: blocks/embed/embed.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/*
```
Comment (RemoLiechti) [inline blocks/embed/embed.js:1]: copied from boilerplate

File: styles/styles.css [hunk 3]
```diff
@@ -26,12 +26,15 @@
   --heading-font-family: 'HCo Ringside Extra Wide', 'HCo Ringside Extra Wide Fallback', sans-serif;
```
Comment (RemoLiechti) [inline styles/styles.css:26]: some of these styles are not yet used in this PR but are referenced already in my working copy, sorry about that.

File: styles/theme-about.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+div.background>div>div>div>div:first-child {
+  background-color: rgb(126 156 191 / 15%);
```
Comment (RemoLiechti) [inline styles/theme-about.css:2]: shall we add a section with section metadata for style?

**Comment by RemoLiechti** [comment]
on the faq-replace, depending on the screen size, the container moves due to different text size. this is part of #66 where I will go through all margins etc in the entire page

**Review by karlpauls — APPROVED** [review]

**Pull request merged**